src/realmGeolContinental.ttl

```diff
@@ -127,12 +132,18 @@ soreagcont:Craton rdf:type owl:Class ;
                               owl:onProperty sorelph:hasPlanetaryStructure ;
                               owl:allValuesFrom soreagcont:Shield
                             ] ;
-            rdfs:label "craton"@en .
+            dcterms:contributor <http://linked.data.gov.au/org/gsq> ;
+            dcterms:source <http://www.glossaryofgeology.org> ;
+            skos:definition "A craton is part of Earth's continental crust that has attained stability and has been little deformed for a prolonged period. Cratons include shield areas, where Precambrian rocks are exposed, and platform areas, where Precambrian rocks are overlain by a thin layer of Phanerozoic strata. (Neuendorf et al., 2011)"@en ;                            
+            skos:prefLabel "craton"@en .
 
 
 ###  http://sweetontology.net/realmGeolContinental/Shield
 soreagcont:Shield rdf:type owl:Class ;
             rdfs:subClassOf soreag:GeologicProvince ;
+            dcterms:contributor <http://linked.data.gov.au/org/gsq> ;
+            dcterms:source <http://www.glossaryofgeology.org> ;
+            skos:definition "A large area of exposed basement rock in a craton, commonly with a very gently convex surface, surrounded by sediment covered platforms. The rocks of virtually all shield areas are Precambrian. (Neuendorf et al., 2011)"@en ;
```

Comment:
I like the revised def. 
Cratonic shields and cratonic platforms are not so fundamentally different, with the distinction being the amount of younger cover on them, or from another perspective, whether the peneplain is exposed (shield) or subsurface (platform).  
I can't give a professional opinion here, or even an educated guess, but I have a feeling is that shields and platforms are parts of a craton rather than kinds of cratons.

Reply:
From what I can gather, a shield is always part of as craton, but if part of the craton is exposed over a large area, that part of it is termed a shield. Please correct me if I am wrong. Online, you will see reference to the Australian Shield (= Western Australian Shield), but, as far as I can determine, GA does not embrace this usage, and all Australian cratonic regions are referred to by them only as ‘cratons’.

The revised definition is little different to the original definition, the exception being the exclusion of the scope note; and all of it is referred to the Glossary (Neuendorf et al.). I am quite happy with this, but the problem is with the referencing, an issue also raised elsewhere here, several times. From a copyright perspective, we can get away with referencing a definition, in this case for a ‘shield’, to the Glossary. But, to provide a direct link for definition of terms (for all terms in a definition), as has been suggested (and clearly a necessity), becomes a problem. We cannot provide links to Neuendorf et al., as it is not freely available. And, from what we have here in all of this feedback, is that their definitions have issues. Apart from directly linking to free, reputable-source definitions (that we like), do we need to develop our own glossary (which also includes and directly acknowledges the liked definitions)? When you look at the comments here, this is what we are doing (i.e., developing our own definitions)!! To base definitions directly on Neuendorf (or other published glossaries) would breach copyright fair-dealing, without a paid agreement/subscription in place. This, however, brings us to the Glossary of geology — Wikipedia:
https://en.wikipedia.org/wiki/Glossary_of_geology

And Wikipedia has also been calling out for monetary contributions too; and, besides, it’s not ‘officially’ published. When I compare the two (obviously not fully), Wikipedia appears to have taken the Glossary definitions and just changed the wording/phraseology, to overcome copyright (at least for the time being). 

In developing vocabs, we have been instructed to follow published national and international standards and published definitions (not make up our own). But, this seems to be almost unworkable, without falling foul of copyright laws and having agreements in place with copyright owners. I can see where those of you that don’t want definitions are coming from, but, without definitions, we have disorder. Maybe, considering the scale of what we are doing, it is possible to develop mutually-beneficial relationships (whatever they could be) with publishers, but, then, as also pointed here, the Glossary is not up-to-date either (as likely would be the case for other glossaries/dictionaries). Considering the latter point, I have been disturbed, confused and frustrated by the distinctly different definitions provided for some terms by various reputable sources; and have been astonished by the lack of definitions provided in major references on a number of subjects, e.g., a recently published book on Coal Geology (that I was looking through) that defines coal, but not coal geology!! This is not an isolated instance.

Precis: Revised definition OK, with SKOS note. But published definitions are largely associated with copyright issues and we seem to have issues with them in any case. It further seems to me that we want to develop our own definitions—big job!!